src/typedKnex.ts

```diff
@@ -1316,6 +1317,11 @@ class TypedQueryBuilder<ModelType, SelectableModel, Row = {}> implements ITypedQ
         }
     }
 
+    public getColumnNameWithQuotes(...keys: string[]): string {
+        const unquotedKeys = this.getColumnName(...keys);
+        return unquotedKeys.split('.').map(col => `"${col}"`).join('.');
```

Comment:
I needed this to get the test with the raw where to work. `getColumnName` doesn't put double quotes around the table name and properties. I bet there's a better way to do this but I'll leave that to you :D

Reply:
I didn't merge latest `master` in to this branch. Now I did, and https://github.com/wwwouter/typed-knex#getcolumnalias now is available.
I think this should do what you need